app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.UserProfileFragment">
+
+
+    <TextView
+        android:id="@+id/settings"
+        android:layout_width="126dp"
+        android:layout_height="47dp"
+        android:layout_marginTop="88dp"
+        android:text="@string/settings_title"
+        android:textColor="@color/seed"
+        android:textSize="30sp"
```

Comment:
Use one of the predefined text sizes or define a new one. Take a look [here](https://github.com/material-components/material-components-android/blob/master/docs/theming/Typography.md)